src/pages/cloud-native-management/kanvas/operator.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+import SEO from "../../../components/seo";
+
+import KanvasOperator from "../../../sections/Kanvas/Kanvas-visualize/index";
```

Comment:
Are we hyphenating or using snake case in our folder names?

Reply:
hyphenating